src/BaseFactory.php

```diff
@@ -58,14 +74,23 @@ public function with(string $relatedModelClass, string $relationshipName, int $t
     {
         $clone = clone $this;
 
+        $forignKey = $this->getForignKeyFromClassName($clone->modelClass);
         $clone->relatedModel = $this->getFactoryFromClassName($relatedModelClass)
             ->times($times)
-            ->make();
+            ->make([
+                $forignKey => null
```

Comment:
Why do you set here the foreign key to here null?

Reply:
Because if I didn't set it to null, and I tried to use 

```
ChapterFactory::new()->with(Page::class,'pages',3)->create();
```

Four chapters will be created because each Page creates a new chapter by default.